client/signup/steps/plans/index.jsx

```diff
@@ -30,12 +30,20 @@ export class PlansStep extends Component {
 	state = {
 		isDesktop: isDesktop(),
 		experiment: null,
+		mobileExperiment: null,
 	};
 
 	componentWillMount() {
 		loadExperimentAssignment( 'disabled_monthly_personal_premium_v2' ).then( ( experimentName ) => {
 			this.setState( { experiment: experimentName } );
 		} );
+		if ( isMobile() ) {
+			loadExperimentAssignment( 'calypso_mobile_plans_page_with_billing' ).then(
```

Comment:
I recommend that we pre-fetch the assignment, ideally in `client/my-sites/checkout/controller.jsx`. Otherwise, users on slow connections might see a flicker on the "billed annually" copy. In the video below, check at 00:04 how the copy goes from `billed annually` to `billed as $96 annually`. An example of pre-fetching is [here](https://github.com/Automattic/wp-calypso/pull/59771/files#diff-9d7d6e5b810ec4d930d0dd4ab43ab0a72e8082274e818ed6888ec938a94dacc2R330).

https://user-images.githubusercontent.com/1269602/149169990-42d9be08-9aa5-4f4d-8465-87a409422d14.mov

Reply:
This should be fixed now.